tests/contextfreeapi.test.ts

```diff
@@ -306,10 +305,9 @@ export class TransitionTests {
 
 async function main() {
   // First hurdle - configuration.
-  const config = generateDBOSTestConfig(); // Optional.  If you don't, it'll open the YAML file...
+  const config = generatePublicDBOSTestConfig();
```

Comment:
test the new switch path when launching -- the other functions in this test take the old path.